graphics/vulkan-validationlayers/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        KhronosGroup Vulkan-ValidationLayers v1.1.112
+name                vulkan-validationlayers
+version             v1.1.112
+
+categories          graphics
+platforms           darwin
+license             Apache-License-2.0
```

Comment:
This is how we specify the Apache 2.0 license:

`license             Apache-2
`